sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_attribution_v1/query.sql

```diff
@@ -1,16 +1,193 @@
+    city,
+    country,
+    first_seen_date,
+    EXTRACT(YEAR FROM first_seen_date) AS first_seen_year,
+    is_default_browser,
+    normalized_app_name,
+    submission_date,
+    days_since_seen,
+    ad_click,
+    organic_search_count,
+    search_count,
+    search_with_ads,
+    uri_count,
+    active_hours_sum
+  FROM
+    `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1`
+  WHERE
+    submission_date = @submission_date
+),
+fenix_first_date_attributions AS (
+  -- Firefox Preview beta
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
+  FROM
+    org_mozilla_fenix.metrics AS org_mozilla_fenix_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND metrics.string.metrics_install_source IS NOT NULL
+    AND metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    1
+  UNION ALL
+  -- Firefox Preview nightly
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
+  FROM
+    org_mozilla_fenix.metrics AS org_mozilla_fenix_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND metrics.string.metrics_install_source IS NOT NULL
+    AND metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    1
+  UNION ALL
+  -- Fenix nightly
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
+  FROM
+    org_mozilla_fennec_aurora.metrics AS org_mozilla_fennec_aurora_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND metrics.string.metrics_install_source IS NOT NULL
+    AND metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    1
+  UNION ALL
+  -- Fenix beta
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
+    ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] AS adjust_network,
+    ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] AS install_source
+  FROM
+    org_mozilla_firefox_beta.metrics AS org_mozilla_firefox_beta_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+    AND metrics.string.metrics_install_source IS NOT NULL
+    AND metrics.string.metrics_adjust_network IS NOT NULL
+  GROUP BY
+    1
+  UNION ALL
+  -- Fenix release
+  SELECT
+    client_info.client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
```

Comment:
this would always be the lowest value found for `submission_timestamp` in the specific date range used? `submission_timestamp` field name could be misleading here.

Reply:
Field name updated to avoid misleading!